Userland/BuggieBox/main.cpp

```diff
@@ -46,12 +46,29 @@ ENUMERATE_UTILITIES(DECLARE_ENTRYPOINT)
 
 static void fail()
 {
-    out(stderr, "Direct running of BuggieBox was detected without finding a proper requested utility.\n");
-    out(stderr, "The following programs are supported: uname, env, lsblk, file, df, mount, umount, mkdir, ");
-    out(stderr, "rmdir, rm, chown, chmod, cp, ln, ls, mv, cat, md5sum, sha1sum, sha256sum, sha512sum, sh, uniq, id, tail, ");
-    out(stderr, "find, less, mknod, ps\n");
-    out(stderr, "To use one of these included utilities, create a symbolic link with the target being this binary, and ensure the basename");
-    out(stderr, "is included within.\n");
+#define TOOL_NAME(name) #name,
+    auto const tool_names = {
+        "md5sum", "sha1sum", "sha256sum", "sha512sum",
+        "Shell", ENUMERATE_UTILITIES(TOOL_NAME)
+    };
+#undef TOOL_NAME
+
+    outln(stderr);
+    outln(stderr, "Usage:");
+    outln(stderr, "1. Create a symbolic link with the target being this binary,");
+    outln(stderr, "   and ensure the basename is one of the supported utility's name.");
+    outln(stderr, "2. Specify a utility as an argument:");
+    outln(stderr, "   $ BuggieBox UTILITY");
+
+    outln(stderr);
+    outln(stderr, "The following utilities are supported:");
+    int count = 0;
+    for (auto const& tool_name : tool_names) {
+        if (++count % 5 == 1)
+            out(stderr, "\n\t");
+        out(stderr, "{:12}", tool_name);
```

Comment:
We don't seem to have support for `{:*}` (where the length is then passed in as the parameter), so nothing that needs to be done here as of now, but having the padding be calculated automatically would be pretty cool to have in the future.

Reply:
I think `{:{1}}` works?

Reply:
But that would be an overkill as we don't account for the terminal width. I selected `12` and `5` because `8(tab width) + 12*5 = 68`, which is the width of the terminal if you snap it to one half of the screen.

Reply:
That's a fairly arbitrary number, since screens come in all kinds of different resolutions, and people often resize their terminals anyways.

If we are trying to take the terminal size into account, we might as well query the actual width. If we don't, we should at least make sure that the table takes the least horizontal space possible (i.e. minimum padding).

Reply:
But we have to stop at some point right? `68` is good enough. (`66` is the width of the longest sentence and i don't think word splitting on sentences is gonna be done). So, for now, `68` is the smallest terminal width you can have that will not break the table formatting.